docs/pages/submit/android.mdx

```diff
@@ -1,78 +1,187 @@
-import { Step } from '~/ui/components/Step';
 
-This guide outlines how to submit your app to the Google Play Store from your own computer and from CI.
+This guide outlines how to submit your app to the Google Play Store from your computer or from a CI/CD service.
+
+## Submitting your app from your computer
+
+<Prerequisites numberOfRequirements={7}>
+  <Requirement number={1} title="Sign up for a Google Play Developer account">
+    A Google Play Developer account is required to submit your app to the Google Play Store. You can sign up for a Google Play Developer account on the [Google Play Console sign-up page](https://play.google.com/apps/publish/signup/).
+  </Requirement>
+  <Requirement number={2} title="Create a Google Service Account">
+    EAS requires you to upload and configure a Google Service Account Key to submit your Android app to the Google Play Store. You can create one with the [uploading a Google Service Account Key for Play Store submissions with EAS](https://github.com/expo/fyi/blob/main/creating-google-service-account.md) guide.
+  </Requirement>
+  <Requirement number={3} title="Create an app on Google Play Console">
+    Create an app by clicking **Create app** in the [Google Play Console](https://play.google.com/apps/publish/).
+  </Requirement>
+  <Requirement number={4} title="Install EAS CLI and authenticate with your Expo account">
+    Install EAS CLI and login with your Expo account:
+    
+    <Terminal cmd={['$ npm install -g eas-cli && eas login']} />
+  </Requirement>
+  <Requirement number={5} title="Include a package name in app.json">
+    Include your app's package name in **app.json**:
+
+    ```json app.json
+    {
+      "android": {
+        "package": "com.yourcompany.yourapp"
+      }
+    }
+    ```
+
+  </Requirement>
+  <Requirement number={6} title="Build a production app">
+    You'll need a production build ready for store submission. You can create one using [EAS Build](/build/introduction/):
+    
+    <Terminal cmd={['$ eas build --platform android --profile production']} />
+
+    Alternatively, you can build the app on your own computer with `eas build --platform android --profile production --local` or with Android Studio.
+
+  </Requirement>
+  <Requirement number={7} title="Upload your app manually at least once">
+    You have to upload your app manually at least once. This is a limitation of the Google Play Store API.
+
+    Learn how with the [first submission of an Android app](https://expo.fyi/first-android-submission) guide.
+
+  </Requirement>
+</Prerequisites>
+
+Once you have completed all the prerequisites, you can start the submission process.
+
+Run the following command to submit a build to the Google Play Store:
+
+<Terminal cmd={['$ eas submit --platform android']} />
+
+The command will lead you step by step through the process of submitting the app. You can configure the submission process by adding a submission profile in **eas.json**:
```

Comment:
```suggestion
The command will lead you step by step through the process of submitting the app. You can configure the submission process by adding a submission profile in **eas.json**:
```

Reading the last sentence here, it is not clear why I need to do this after running the command. Should this step of adding `serviceAccountKeyPath` mentioned before running the command?

Reply:
It's not required to do before running `eas submit`, it's just a shortcut through the prompts. My goal here was to show the interactive command you can just run, but also share that you can provide info to make it faster. Lmk if you think there's a better way to present that info, or if i should just omit part of this.

Reply:
Thanks for the context!

I have made a small suggestion to explain the "why" of adding the eas.json configuration. This was the reason it was confusing in the first place because the lack of explanation why I'd need to add `serviceAccountKeyPath`. I think we can just tell the developers about that, and it'll avoid confusion.